.carousel test.html

```diff
@@ -0,0 +1,62 @@
+.carousel {
```

Comment:
I guess this is some kind of the test file that you should get rid of :D

Reply:
How? Because i dont know how to delete this